js/front/dataset/index.js

```diff
@@ -72,21 +71,15 @@ new Vue({
         },
 
         /**
-         * Extract the current dataset metadatas from microdata markup
+         * Extract the current dataset metadatas from JSON-LD script
          * @return {Object} The parsed dataset
          */
         extractDataset() {
-            const dataset = microdata('http://schema.org/Dataset')[0];
-            if (Array.isArray(dataset.distribution)) {
-                dataset.resources = dataset.distribution;
-            } else {
-                dataset.resources = dataset.distribution ? [dataset.distribution] : [];
-            }
+            const selector = "script[type='application/ld+json']";
```

Comment:
Maybe we can speed this a little bit by using an ID.
Having this helper outside of the view would allows to test it and reuse this mecanic in other views.